### PR TITLE
@nico/avatars in contacts

### PR DESCRIPTION
**Pull request by nicoglennon**
Adds new color/emoji support for contacts. Fixes RNBW-1349.

PoW: https://www.loom.com/share/b6bbd49f6df7427eb3ffaca009081b34

Contacts stored as ens domains would stay in their letter form (as shown in the video by nico.eth) if the migration happens while the user is not connected to the internet.

### CODE REVIEW

**Comment by nicoglennon** [comment]
@mikedemarais @skylarbarrera thoughts / comments on the migration portion? If the user is offline the moment the migration runs, if they have any .eth contacts those will not get updated to emojis since we'd need to resolve the names to addresses via an ethers.js call... so those get skipped. Any ideas on how to avoid this?

File: src/components/send/SendHeader.js [hunk 1]
```diff
@@ -74,12 +78,24 @@ export default function SendHeader({
...
   }, [contacts, recipient]);
+  const [hexAddress, setHexAddress] = useState(null);
+
+  useEffect(() => {
+    if (isValidAddress) {
+      getAddress();
+    } else {
+      setHexAddress(null);
+    }
+    async function getAddress() {
```
Comment (brunobar79) [inline src/components/send/SendHeader.js:91]: The name here is confusing. It's called get but it actually sets in state

**Review by brunobar79 — APPROVED** [review]
💯

**Review by 0racl3z — COMMENTED** [review]
thx!

**Pull request merged**